src/test/testGraphs.ts

```diff
@@ -1,18 +1,32 @@
-import { Graph, HexString } from "../utilities/types";
+import { DSNPUserId } from "@dsnp/sdk/dist/types/core/identifiers";
+import { HexString } from "../utilities/types";
 import { generateDsnpUserId, getPrefabDsnpUserId } from "./testAddresses";
 
+type Graph = Record<DSNPUserId, Record<DSNPUserId, boolean>>;
+
+interface SocialGraph {
+  followers: Graph;
+  following: Graph;
+}
+
 export const generateRandomGraph = (
   dsnpUserId: HexString,
-  size: number = 4
-): Graph => {
-  const following = [...Array(size)].map(() => generateDsnpUserId());
-  const followers = [...Array(size)].map(() => generateDsnpUserId());
+  _size: number = 4
+): Record<DSNPUserId, boolean> => {
+  return {};
+};
 
-  return {
-    dsnpUserId,
-    following,
-    followers,
-  };
+const followersFromFollowing = (following: Graph): Graph => {
```

Comment:
What is the purpose of this function?

Reply:
Given a map of accounts to a map of the accounts they follow, this inverts the map into a map of followed accounts to a map of accounts that follow them. I'll document this in the next PR.